src/averageAges.js

```diff
@@ -37,7 +51,18 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const motherList
+  = people.filter(human => human.mother).map(human => human.mother);
+
+  const womanList = withChildren
+    ? people.filter(
+      human => human.sex === 'f' && motherList.includes(human.name))
```

Comment:
```suggestion
    ? people.filter((human) => (
      human.sex === 'f' && motherList.includes(human.name)
    ))
```
try to avoid such transferring as it was